sdk/tests/core/inc/az_test_log.h

```diff
@@ -6,10 +6,10 @@
 
 // This macro helps with the expected values for tests when verifying logging.
 
-#ifndef AZ_NO_LOGGING
-#define _az_BUILT_WITH_LOGGING(value_if_yes, value_if_no) (value_if_yes)
-#else
+#ifdef AZ_NO_LOGGING
```

Comment:
I strongly recommend to revert it back, as I said above.

Reply:
Now I don't feel that strong, but I still like the previous order and optimizing for happy path first, it is better from the reader's perspective, IMO.